docs/migration-guide.md

```diff
@@ -0,0 +1,37 @@
+# 📙 BotD API Migration Guide
+
+The API has changed, and **you need to update your code**.
+
+In the new version, we have implemented detection logic on the client side, meaning we don't make requests to our servers. Also you don't need to generate any API keys anymore.
+
+## Previous `0.1`
+
+```ts
+const botdPromise = import('https://openfpcdn.io/botd/v0.1').then((Botd) =>
+    Botd.load({ publicKey: '<your-public-key>' }),
+)
+botdPromise
+    .then((botd) => botd.detect())
+    .then((result) => console.log(result))
+    .catch((error) => console.error(error))
+```
+
+## Current
+
+```ts
+import { load } from '@fpjs-incubator/botd-agent'
```

Comment:
Changing the installation method during migration is not necessary and may confuse the readers.

A good practice is providing a migration guide for every installation method that was recommended in the previous version.

Consider using the `diff` highlighting to make the code changes clearer. Example:

```diff
  foo(
-   'bar'
+   'baz'
  )
```